src/test/java/com/cloudant/tests/CloudantClientTestBase.java

```diff
@@ -0,0 +1,82 @@
+package com.cloudant.tests;
+
+import com.cloudant.client.api.CloudantClient;
+
+import org.junit.After;
+import org.junit.Before;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+/**
+ * Created by Rhys Short on 21/05/15.
+ */
+public abstract class CloudantClientTestBase {
```

Comment:
Not quite sure I've understood about making it less painful if we change the constructor for `CloudantClient`, as if we return a `CloudantClient` from a static method in the helper class, it's still only in the one place where we'd have to change it. My inclination would be to make it a helper class.